privacyidea/lib/smsprovider/FirebaseProvider.py

```diff
@@ -95,6 +95,8 @@ class FIREBASE_CONFIG:
     API_KEY = "apikey"
     APP_ID_IOS = "appidios"
     API_KEY_IOS = "apikeyios"
+    HTTP_PROXY = "httpproxy"
```

Comment:
This setting is not necessary, since the firebaseprovider always connects via HTTPS.